cmd/drand-cli/dkg_cli.go

```diff
@@ -257,17 +258,27 @@ func parseInitialProposal(c *cli.Context) (*drand.FirstProposalOptions, error) {
 		return nil, err
 	}
 
+	period := c.Duration(periodFlag.Name)
 	timeout := time.Now().Add(c.Duration(dkgTimeoutFlag.Name))
-	transitionTime := time.Now().Add(c.Duration(transitionTimeFlag.Name))
+
+	// if a custom time hasn't been added, we set a one based on the period
+	// to make the tests a little more predictable
+	var genesisTime time.Time
+	if c.IsSet(transitionTimeFlag.Name) {
+		genesisTime = time.Now().Add(c.Duration(transitionTimeFlag.Name))
+	} else {
+		defaultRoundsToWait := period * 3
+		genesisTime = time.Now().Add(defaultRoundsToWait)
+	}
```

Comment:
Yeah it kinda is - I'm not sure what would make a more sensible default. If I set it much longer (which is reasonable for prod) then it'll be way too long for the tests or people who just want to spin up a local network for trying out drand (although the tests ought to pass their own [though live envs ought to pass their own too!])

Reply:
Okay have made the transition time mandatory - turns out the tests already pass it. We will need to rethink magically bringing it forward in case of quick DKGs